src/common/errors.py

```diff
@@ -7,3 +7,7 @@
 
 class NotCoordArrayError(TypeError):
     """Raised if the expected coord object is not a coord object"""
+
+
+class ConfigAttributeNotFound(ValueError):
```

Comment:
Would it be best to explain the structure of the `img_coords.json` within the `ConfigAttributeNotFound` class? The `KeyError` exception is a viable choice in the case of dictionaries. I feel providing a custom error indicating what you're trying to access may improve readability in terms of debugging. For example, if `ConfigAttributeNotFound` error is raised, then it will automatically indicate that this is a `img_coords.json` issue.

Reply:
Sounds good, thanks for the explanation here! Maybe a reference to the file you mentioned within the docstring could help. Please consider this optional / up to your best judgement.